pop-api/src/lib.rs

```diff
@@ -33,6 +33,16 @@ mod constants {
 	pub(crate) const BALANCES: u8 = 10;
 }
 
+/// Helper method to build `ChainExtensionMethod``
+pub fn build_extension_method(
```

Comment:
Don't expect this to be used outside the crate so pub unnecessary for now. We want devs to use the API functions, hiding this complexity from them.